addons/main/script_macros.hpp

```diff
@@ -60,3 +60,9 @@
         backpack = #BACKPACK; \
         count = COUNT; \
     }
+
+#define MACRO_SET_VEHICLE_CARGO \
+        class TransportBackpacks {}; \
+        class TransportItems {MACRO_ADDITEM(FirstAidKit,4)}; \
+        class TransportMagazines {}; \
+        class TransportWeapons {};
```

Comment:
I would try to make it similar to other vanilla vehicles, so some weapon (from Theseus Services ideally), a GPS and whatever else is there.

Rename macro to indicate land vehicle, as air would have parachutes in addition.